recipe/conda_build_config.yaml

```diff
@@ -1,68 +1,18 @@
 python_abi_tag:
- - cp38
- - cp39
- - cp310
- - cp311
- - cp312
  - cp313
  - cp313t
- - pypy38_pp73
- - pypy39_pp73
- - graalpy230_310_native
- - graalpy231_310_native
 
 python_impl_version:
- - 3.8
- - 3.9
- - 3.10
- - 3.11
- - 3.12
  - 3.13
  - 3.13
- - 7.3
- - 7.3
- - 23.0
- - 23.1
 
 python_version:
- - 3.8
- - 3.9
- - 3.10
- - 3.11
- - 3.12
```

Comment:
All these python variants are unnecessary, and this feedstock is designed only for **py313**?

Reply:
Correct, since our previous python versions do not depend on python_abi.